base_layer/wallet/src/utxo_scanner_service/utxo_scanner_task.rs

```diff
@@ -325,7 +325,11 @@ where
         client: &mut BaseNodeWalletRpcClient,
     ) -> Result<BlockHeader, UtxoScannerError> {
         let tip_info = client.get_tip_info().await?;
-        let chain_height = tip_info.metadata.map(|m| m.height_of_longest_chain()).unwrap_or(0);
+        let wallet_birthday_height = self.get_birthday_header_height_hash(client).await?.height;
```

Comment:
The old code is correct. 
If the wallet cant get the network tip from the base_node its connected to it needs to assume its 0. 
It also very unlikely that wallet wont have a tip, but has random height header?

Reply:
Great point ! You are right :)